doc/api/modules.md

```diff
@@ -600,6 +600,16 @@ filename scales linearly with the number of registered extensions.
 In other words, adding extensions slows down the module loader and
 should be discouraged.
 
+#### require.main
+<!-- YAML
+added: v0.1.17
+-->
+
+* {Object}
+
+The main module of the entry script.
```

Comment:
The description isn't overly useful. Perhaps something like...

```
The `Module` object representing the entry script loaded when
the Node.js process launched.
```

Then include a quick example:

```sh
$ node entry.js
```

```js
console.log(reqiure.main)
// Prints
// Module {
//   id: '.',
//   exports: {},
//   parent: null,
//   filename: '/absolute/path/to/entry.js',
//   loaded: false,
//   children: [],
//   paths:
//    [ '/absolute/path/to/node_modules',
//      '/absolute/path/node_modules',
//      '/absolute/node_modules',
//      '/node_modules' ] }
```

Reply:
Updated.